### PR TITLE
Fix CRDs creation + fix credential de-provisioning + logging

### PR DESCRIPTION
**Pull request by dgghinea**
- updated gitignore
- fixed failing status deprovision
- fixed extra CRDs being created

### CODE REVIEW

**Review by alrosca — APPROVED** [review]

File: pkg/subscription/credential/credential.go [hunk 1]
```diff
@@ -158,55 +173,68 @@ func (p credentialProvisioner) Update() (provisioning.RequestStatus, provisionin
 		return rs.SetMessage("kongCredentialId cannot be empty").Failed(), nil
 	}
 
+	p.logger.WithField("credentialID", credentialID).
```
Comment (jcollins-axway) [inline pkg/subscription/credential/credential.go:176]: WithField does not update the logger, it returns a new logger. If you want to add these fields to all log lines in this method use

```
log := p.logger.WithField("credentialID", credentialID).WithField("consumerID", consumerID)

log.Info("Started credential update")

File: pkg/subscription/credential/credential.go [hunk 2]
```diff
@@ -93,6 +100,8 @@ func (p credentialProvisioner) Provision() (provisioning.RequestStatus, provisio
 	ctx := context.Background()
 	rs := provisioning.NewRequestStatusBuilder()
 	credentialType := p.request.GetCredentialType()
+	p.logger.WithField("consumerID", consumerID)
```
Comment (jcollins-axway) [inline pkg/subscription/credential/credential.go:103]: See comment on line 176

File: pkg/subscription/credential/credential.go [hunk 3]
```diff
@@ -56,27 +57,33 @@ func (p credentialProvisioner) Deprovision() provisioning.RequestStatus {
 	if credentialID == "" {
 		return rs.SetMessage("CredentialID cannot be empty").Failed()
 	}
+	p.logger.WithField("credentialID", credentialID).
```
Comment (jcollins-axway) [inline pkg/subscription/credential/credential.go:60]: comment on 176

File: pkg/subscription/credential/credential.go (hunk 1 above)
Comment (dgghinea) [reply pkg/subscription/credential/credential.go:176]: oh. silly me

**Comment by jcollins-axway** [comment]
update the title to something more meaningful, please.

**Review by jcollins-axway — APPROVED** [review]

**Pull request merged**